API.md

```diff
@@ -0,0 +1,235 @@
+# API documentation
+
+## ToC
+
+- [`benmvp create`](#benmvp-create)
+  * [Manual installation](#manual-installation)
+- [`benmvp test`](#benmvp-test)
+- [`benmvp start`](#benmvp-start)
+- [`benmvp build`](#benmvp-build)
```

Comment:
What do you think about have this as an overview for each. then their own doc page for each with more examples and their flags.

Reply:
Like each overview would be 
```
benmvp create
This does a thing, and this is what you expect it to do.
*How to use*
`benmvp create`
Link to read more (examples, arguments)
```

Reply:
I really like this idea of having a separate doc page for each command. There's enough details now that it warrants that. Will do that in my next PR. Thanks!